Lib/fontbakery/cli.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import absolute_import, print_function, unicode_literals
+
+import argparse
+import importlib
+import sys
+
+
+def main(args=None):
+    subcommands = {
+        "check-googlefonts": "fontbakery.commands.check_googlefonts",
+        "check-specification": "fontbakery.commands.check_spec",
+        "check-ufo-sources": "fontbakery.commands.check_ufo_sources"
+    }
+
+    if len(sys.argv) >= 2 and sys.argv[1] in subcommands:
+        # Relay to subcommand.
+        subcommand = subcommands[sys.argv[1]]
+        sys.argv[0] += " " + sys.argv[1]
+        del sys.argv[1]  # Make this indirection less visible for subcommands.
+        cmd = importlib.import_module(subcommand)
+        sys.exit(cmd.main())
```

Comment:
instead of `importlib.import_module` and then `sys.exit(cmd.main())`, you could use `runpy.run_module` as in:
https://github.com/fonttools/fonttools/blob/88f495dd68a20c1e16043e57e072324c628a8dc9/Lib/fontTools/__main__.py#L28-L29

https://docs.python.org/3/library/runpy.html#runpy.run_module

Reply:
that's basically like doing `python -m some_module`